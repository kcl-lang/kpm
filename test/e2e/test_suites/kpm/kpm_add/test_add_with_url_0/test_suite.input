kpm add oci://localhost:5001/test/helloworld?tag=0.1.1