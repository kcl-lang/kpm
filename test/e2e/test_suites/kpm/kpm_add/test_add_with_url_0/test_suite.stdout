add dependency 'helloworld:0.1.1' successfully